### PR TITLE
Service worker in SDK

### PR DESCRIPTION
**Pull request by DanielBrookRoberge**
This PR switches the Merlins worker to use the central worker code in the SDK. Currently that is in PR https://github.com/mobify/progressive-web-sdk/pull/549/; until we merge, release, and update to the new release, this will fail on Circle

 **JIRA**: (link to JIRA ticket)
 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/549/

## Changes
- Use the SDK worker 

## How to test-drive this PR
- Install the associated SDK branch
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- See that a new service worker is registered with version 0.2.0 (as printed in the console)
- Reload the home page twice
- See that all three caches are populated
- Go offline
- See that the home page loads again.

### CODE REVIEW

File: web/worker/main.js [hunk 1]
```diff
@@ -1,103 +1,12 @@
 /* global PROJECT_SLUG, DEBUG */
 /* eslint-env worker, serviceworker */
 
-import toolbox from 'sw-toolbox'
+import worker from 'progressive-web-sdk/dist/worker/main'
 
-const cachebreaker = /b=([^&]+)/.exec(self.location.search)[1]
 const CAPTURING_URL = 'https://cdn.mobify.com/capturejs/capture-latest.min.js'
```
Comment (jnurse) [inline web/worker/main.js]: This looks like it's the same URL the worker in the SDK is using as a default. Is this meant to be an example of how you can pass your own capturing URL into the worker?

Comment (DanielBrookRoberge) [inline web/worker/main.js]: Yes. I'm not 100% sure I still want to keep the default in the SDK, either.

File: web/worker/main.js [hunk 2]
```diff
@@ -1,103 +1,12 @@
...
-    e.waitUntil(
-        self.clients.claim()
-            .then(() => caches.keys())
-            .then((cacheKeys) => cacheKeys.filter(
-                (key) => cacheNames.indexOf(key) === -1 && !key.endsWith('$$$inactive$$$')))
-            .then((keysToDelete) => keysToDelete.map((key) => caches.delete(key)))
-            .then((promises) => Promise.all(promises))
-    )
+worker({
+    capturingURL: CAPTURING_URL,
```
Comment (jeremywiebe) [inline web/worker/main.js]: Given we have this defaulted, would it be clearer to not "override" the capturing URL here?

File: web/worker/main.js (hunk 1 above)
Comment (jeremywiebe) [inline web/worker/main.js]: I _think_ we want to keep the default in the SDK ... but I'd vote to remove the "override" here.

File: web/worker/main.js [hunk 3]
```diff
@@ -1,103 +1,12 @@
...
+    slug: PROJECT_SLUG,
+    debug: DEBUG
```
Comment (jeremywiebe) [inline web/worker/main.js]: I believe this is `isDebug` in the SDK.

**Pull request merged**